libheif/plugins/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ macro(plugin_compilation name varName optionName defineName)
                     LIBHEIF_EXPORTS
                     HAVE_VISIBILITY)
             target_compile_definitions(heif-${name} PRIVATE PLUGIN_${defineName}=1)
-            target_include_directories(heif-${name} PRIVATE ${PROJECT_SOURCE_DIR} ${libheif_BINARY_DIR} ${libheif_SOURCE_DIR} ${${varName}_INCLUDE_DIR})
+            target_include_directories(heif-${name} PRIVATE ${PROJECT_SOURCE_DIR} ${libheif_BINARY_DIR} ${libheif_SOURCE_DIR} ${${varName}_INCLUDE_DIR} ${${varName}_INCLUDE_DIRS})
```

Comment:
Perhaps we should simply change `${varName}_INCLUDE_DIR` to `${varName}_INCLUDE_DIRS` instead of adding?

Reply:
> If any of the libheif supplied custom FindXYZ.cmake modules does not produce the `XYZ_INCLUDE_DIRS` variable, [it should be fixed there](https://cmake.org/pipermail/cmake/2006-October/011502.html).

Indeed. Please fix those in all cmake/modules/FindXYZ.cmake modules: the `libfind_process(XYZ)` produces those, and they should be used in all subsequentl `find_package_handle_standard_args(XYZ)` calls.

Reply:
Thank you for your feedback, I will study up on this issue.

Reply:
I simply removed `${varName}_INCLUDE_DIRS` which restores those two lines to code to how they originally were. It turns out that they weren't necessary.

Reply:
It should be "fixed" anyway, but that's maybe for a separate PR.

Reply:
@kmilos Yes, but let's do this in a different issue so that @dukesook can concentrate on the JPEG2000 parts. (#917)